src/containers/costume-library.jsx

```diff
@@ -26,10 +26,13 @@ class CostumeLibrary extends React.PureComponent {
         ]);
     }
     handleItemSelected (item) {
+        const type = item.md5.split('.')[1];
```

Comment:
Hmmm it's a little dangerous to assume that String.split yields a 2 element array.